website/blog/2023-02-03-product-update.md

```diff
@@ -0,0 +1,76 @@
+
+## **What's Better?**
+
+- **[The new dbt Cloud IDE](https://www.getdbt.com/blog/new-improved-cloud-ide/)** is live, and available to everyone. It’s faster, more intuitive, and includes some neat quality-of-life features like git diff view, autocomplete, a code formatter, and an improved syntax highlighting.
+- **[Custom node colors in the dbt DAG](https://www.getdbt.com/blog/announcing-dbt-v1.3-and-utils/).** Color coding your DAG makes it *much* easier to understand your project at a glance. Read about this update, as well as other dbt Core v1.3 improvements.
+- **[New Service Bundles](https://www.getdbt.com/blog/revamped-professional-services):** The dbt Labs Pro Services team is re-launching its services offerings with new bundles and engagement models, making it simpler for customers to find an engagement scope that works for them.
+- **Removing defaults:** “Run on schedule” is no longer the default setting when creating a new job definition in the dbt Cloud UI. No more accidentally scheduled jobs + unwanted runs.
+
+## **New Resources**
+
+*Check out the **[dbt Developer Hub](https://docs.getdbt.com/)** for all-in-one location to learn, engage with peers, and contribute to the craft of analytics engineering.*
+
+- **[How we cut our tests by 80% while increasing data quality: the power of aggregating test failures in dbt](https://docs.getdbt.com/blog/aggregating-test-failures-with-dbt)** by Noah Kennedy at Tempus
+- **[How to design and structure dbt metrics: Recommendations for getting started](https://docs.getdbt.com/blog/how-to-design-and-structure-metrics)** by Callum McCann, dbt Labs
+- **[Auto-scaling CI: The intelligent CI](https://docs.getdbt.com/blog/intelligent-slim-ci)** by Doug Guthrie, dbt Labs
+- **[Spin up a dbt+Postgres environment](https://discourse.getdbt.com/t/spin-up-a-dbt-postgres-environment-with-a-single-click-in-vs-code-for-demos-and-interview-assignments/5285/1)** with Panasenco’s template for creating quick demo environments via VS Code.
+- **[Query the dbt Semantic Layer via dbt](https://github.com/Delphi-Data/dbt_natural_language)** with the Delphi API.
+- **[Generate some dogs](https://www.loom.com/share/e33fdfa062dd4764b828002f6cffbf6a)** in the IDE, on the house. 🐶
+
+![https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png](https://hs-8698602.f.hubspotemail.net/hub/8698602/hubfs/Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png?upscale=true&width=600&upscale=true&name=Screen%20Shot%202023-02-01%20at%208.12.43%20PM.png)
```

Comment:
This is a pretty low-quality screenshot; can you potentially upload it as an actual image into the docs repo and use  a lightbox element (that way you can add alt text description to this)? See D[oug's autoscaling CI blog post for an example](https://raw.githubusercontent.com/dbt-labs/docs.getdbt.com/current/website/blog/2023-01-27-autoscaling-ci.md).